packages/network-score/src/Scoring.ts

```diff
@@ -458,10 +455,42 @@ export async function buildFromRevokeRatingProperties(
   }
 }
 
+/**
+ * Constructs a revised entry for a previously amended rating on the blockchain.
+ * 
+ * This asynchronous function is responsible for building a rating object from revised rating properties. 
+ * It takes a rating entry, chain space URI, and author URI as parameters and returns a promise resolving 
+ * to an object containing the URI of the rating entry and its details. It verifies the signature of the amended rating, 
+ * validates required fields, and generates the necessary signatures for the revised entry. This is essential 
+ * for maintaining the integrity and trustworthiness of the rating system, especially in decentralized environments.
+ *
+ * This function is specifically designed for use by Rating Authors or Ledger/API Providers.
+ * 
+ * @param rating - The rating revise entry to process.
+ *                             This includes the original rating's digest, signature, and other relevant details.
+ * @param chainSpace - The identifier of the blockchain space (as a URI) where the rating is being revoked.
+ *                            This helps in pinpointing the exact location on the blockchain where the rating resides.
+ * @param authorUri - The Decentralized Identifier (DID) URI of the author who is revoking the rating.
```

Comment:
same here - revoke or revise?

Reply:
You are right. Revised is the correct word.